src/shared.js

```diff
@@ -134,7 +137,7 @@ export function foldLine (str, maxLen = 76) {
       if ((match = /.*\s+/.exec(curLine)) && /\S/.test(match[0])) {
         // use everything before and including the last white space character (if anything)
         curLine = match[0];
-      } else if ((match = /.*[\x21-\x2f0-9\x5b-\x60\x7b-\x7e]+/.exec(curLine)) && /[^\x21-\x2f0-9\x5b-\x60\x7b-\x7e]/.test(match[0])) {
+      } else if (!escaped && (match = /.*[\x21-\x2f0-9\x5b-\x60\x7b-\x7e]+/.exec(curLine)) && /[^\x21-\x2f0-9\x5b-\x60\x7b-\x7e]/.test(match[0])) {
```

Comment:
I don't know if this is the right thing to do, but this was matching the forward slash and I don't know why. We need more information about what this condition should be checking for, and figure why its matching a backslash character.

Reply:
I don't know why this should match.

```js
'a\\b'.length;
// => 3
match = /.*[\x21-\x2f0-9\x5b-\x60\x7b-\x7e]+/.exec('a\\b');
match[0];
// => 'a\\'
```

Examination of the escapes as Buffers.

```js
Buffer.from('\\');
// <Buffer 5c>
Buffer.from('\x21-');
// <Buffer 21 2d>
Buffer.from('\x2f0');
// <Buffer 2f 30>
Buffer.from('\x5b-');
// <Buffer 5b 2d>
Buffer.from('\x60');
// <Buffer 60>
Buffer.from('\x7b-');
// <Buffer 7b 2d>
Buffer.from('\x7e');
// <Buffer 7e>
```

Reply:
yes I agree also because it is not clear what this code is for, I think in order not to break the lines maybe if there is a consecutive '\' and '\n' immediately after it?

`/.*[\x21-\x2f0-9\x5b-\x60\x7b-\x7e]+/.exec('a\\b')`
match because 
`/.*`
because this match the 'a'
`[\x21-\x2f0-9\x5b-\x60\x7b-\x7e]+`
and  this the \ and the `+` matches the previous token between one and unlimited times